docs/_tooling/extensions/score_metamodel/metamodel.yaml

```diff
@@ -3,6 +3,20 @@
 needs_types_base_options:
   optional_options:
     source_code_link: "^https://github.com/eclipse-score/score/blob/.*$"
+  # Custom semantic validation rules
+  custom_validation:
+    stop_words:
+      - shall
+      - must
+      - will
+    weak_words:
```

Comment:
What is "weak"? Those are stop words ("prohibited" would be clearer) for the content?!